src/scheduleBot_Manager.py

```diff
@@ -86,16 +89,15 @@ def __init__(self, curr_path, config_path):
             and the second one is a text with response
         """
         new_group = re.sub('[-]', '', new_group)
-        response = list()
         if self.xls.find_timetable(new_group, dict()) != dict():
-            response = [True, f"Група була змінена на {new_group}"]
+            response = True, f"Група була змінена на {new_group}"
```

Comment:
It that case it's better to explicitly set response variable to tuple for it to be more readable